stack/templates/deployment.yaml

```
@@ -71,9 +71,15 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-          {{- with .Values.volumeMounts }}
+          {{- if or .Values.persistence.enabled .Values.volumeMounts}}
```

I think `values.yaml` either needs a default mount path or this needs to check that the user sets a value for it
```suggestion
          {{- if or (and .Values.persistence.enabled .Values.persistence.mountPath) .Values.volumeMounts}}
```